andela/travel_tool_front
# #161962033 Fix checkbox and multi-trip cancel icon designs in request modal

#### What does this PR do?

- Fix checkbox and multi-trip cancel icon designs in request modal
#### Description of Task to be completed?
- fix checkbox request modal to match mockup
- fix cancel icon in multi-trip request to match mockup

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Change into the project directory
- git checkout bg-fix-request-modal-design-161962033
- yarn start
- click new request
- check the alignment of the send request 
- check the cancel icon in multi-trip
- check the mockups

#### Any background context you want to provide?
[Request Form Mockup](https://projects.invisionapp.com/d/#/console/14861108/319652031/preview)
[Multi-trip cancel icon mockup](https://projects.invisionapp.com/d/#/console/14861108/319652029/preview)

#### What are the relevant pivotal tracker stories?
[#161962033](https://www.pivotaltracker.com/story/show/161962033)

#### Screenshots (if appropriate)
<img width="1433" alt="screen shot 2018-11-15 at 00 46 25" src="https://user-images.githubusercontent.com/32410518/48514702-03a61f80-e870-11e8-8ac0-163ab1bab655.png">

<img width="1350" alt="screen shot 2018-11-15 at 20 38 46" src="https://user-images.githubusercontent.com/32410518/48570926-8d102d00-e916-11e8-9e40-b27ff2f48753.png">


#### Questions:
